app/services/listing_index_service/search/converters.rb

```diff
@@ -1,7 +1,7 @@
 module ListingIndexService::Search::Converters
   module_function
 
-  def listing_hash(l, includes)
+  def listing_hash(l, includes, meta={})
```

Comment:
Meta is related to zappy's extra info (like distance). I thought, overloading this function might make sense. This is called also from sphinx side, so the other option is to include empty meta to every call.